css-canvas-medical/styles.css

```diff
@@ -0,0 +1,264 @@
+* {
+  margin: 0 auto;
+}
+
+h1 {
+  font-family: 'Tiempos Headline', sans-serif;
+  font-style: normal;
+}
+
+.header {
+  color: darkblue;
+}
+
+.top-container {
+  display: flex;
+  align-items: center;
+  margin-top: 10px;
+  max-width: 1000px;
+}
+
+.right-container {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 500px;
+}
+
+.top-right {
+  font-family: 'Avenir', sans-serif;
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  word-spacing: 10px;
+  margin-right: 10px;
+  font-size: 10px;
+  color: gray;
+}
+
+.top-button {
+  background-color: rgb(29, 41, 81);
+  color: white;
+  text-decoration: none;
+  font-size: 10px;
+  padding: 8px 13px 8px 13px;
+  border-radius: 4px;
+}
+
+hr {
+  width: 100%;
+  margin-top: 10px;
+  height: 2px;
+  background-color: #666;
+  opacity: 20%;
+}
+
+.first-section {
+  background-image: url('images/hero.jpg');
+  background-size: cover;
+  height: 500px;
+  position: relative;
+  margin-top: 50px;
+  max-width: 1000px;
+}
+
+.overlay {
+  background: black;
+  opacity: 50%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.inside {
+  padding: 90px 0 0 60px;
+  border: 500px;
+}
+
+.text-under-head {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+  margin-left: 40px;
+  width: calc(100% / 2);
+  position: relative;
+  font-size: 12px;
+  line-height: 25px;
+  z-index: 1;
+}
+
+.emr-head {
+  font-family: 'Tiempos Headline', sans-serif;
+  color: white;
+  margin-left: 40px;
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 1;
+}
+
+.middle-button {
+  font-family: 'Avenir', sans-serif;
+  margin-left: 40px;
+  margin-top: 20px;
+  background-color: rgb(29, 41, 81);
+  color: white;
+  font-size: 18px;
+  border-radius: 6px;
+  width: 200px;
+  height: 50px;
+  z-index: 1;
+  position: relative;
+}
+
+.back-ground {
+  background: rgb(250, 243, 243);
+  background-size: cover;
+  max-width: 1100px;
+}
+
+.second-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-top: 80px;
+  color: rgb(59, 64, 78);
+}
+
+.second-head {
+  font-family: 'Tiempos Headline', sans-serif;
+  margin-top: 60px;
+}
+
+.second-text {
+  font-family: 'Avenir', sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.two-columns {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 80px;
+  margin-bottom: 70px;
+  line-height: 60px;
+  text-align: center;
+  color: rgb(61, 74, 116);
+  font-family: 'Avenir', sans-serif;
+  font-size: 14px;
+  padding-bottom: 50px;
+}
+
+.bottom-left-header > * {
+  display: flex;
+  font-family: 'Tiempos Headline', sans-serif;
+  color: rgb(61, 74, 116);
+  max-width: 1000px;
+}
+
+.text-in-last {
+  font-family: 'Avenir', sans-serif;
+  line-height: 30px;
+  font-size: 10px;
+  color: rgb(61, 74, 116);
+  max-width: 1000px;
+}
+
+.bottom-right-container > * {
+  display: flex;
```

Comment:
This seems to be setting flex on an element which has no children, which would mean any flex properties would have no effect. Make sure your selectors are selecting the right elements.